docs/cloud-integration.md

```diff
@@ -3,6 +3,21 @@ layout: global
 displayTitle: Integration with Cloud Infrastructures
 title: Integration with Cloud Infrastructures
 description: Introduction to cloud storage support in Apache Spark SPARK_VERSION_SHORT
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
```

Comment:
There is another ASF license starting in the line 23 as a HTML comment. Is that fine?